api/schema/schema_validator.go

```diff
@@ -24,11 +24,12 @@ func NewSchemaValidator(t *testing.T, schemaName string, got interface{}) *Schem
 	if err != nil {
 		t.Fatal(err)
 	}
+	// Platform compatibility: use "/" separators always for file://
+	dir = strings.Replace(dir, "\\", "/", -1)
```

Comment:
Hmm, I disagree. The specific case it's dealing with is the separators being wrong on Windows. The early-out inside the Go code (should you dig into it) isn't the point, it's that generalising it via `os.PathSeparator` obfuscates the purpose of it in our code. I prefer clarity over no-op generalisation.